### PR TITLE
Migration guide for version 2 #35

### PR DESCRIPTION
**Pull request by poi33**
Symbolic migration guide for version 2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,59 @@
 # Robots.txt
 
-Perhaps the smallest XP app yet? This app has one setting: A TextArea for the entire contents of your robots.txt file (added as an app setting on the sites it is added to). This app will also add a [controller mapping](http://xp.readthedocs.io/en/6.5/developer/site/mappings/index.html#controller-mappings) to the path `robots.txt` on the root of any site it is added to.
+This app allows you to set all fields for the robots.txt file.
+This app will also add a [controller mapping](http://xp.readthedocs.io/en/6.5/developer/site/mappings/index.html#controller-mappings) to the path `robots.txt` on the root of any site it is added to.
```
Comment (rymsha) [inline README.md:4]: time to remove reference to 6.5 docs?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,59 @@
...
 
-No other configurations needed. But be sure to check out [how robots.txt works](https://en.wikipedia.org/wiki/Robots_exclusion_standard) before typing away into the robots-field.
+## Headless
+Headless sites can use the service to get the robots.txt file or data.
+Service path `_/service/com.enonic.app.robotstxt/robotstxt`.
+Accept header of `application/json` will return the data as json.
```
Comment (rymsha) [inline README.md:9]: perhaps describe what happens if `text/plain` is requested

File: README.md [hunk 3]
```diff
@@ -1,24 +1,59 @@
...
+The old configuration is not compatible with the new one, so you need to reconfigure the app.
+
+Exampel 1.0.0 configuration
+```
+Sitemap: https://site.com/sitemap.xml
+User-agent: *
+Disallow:
+```
+
+Migrate this to the respective fields in the new configuration
```
Comment (rymsha) [inline README.md:29]:
```suggestion
Migrate this to the respective fields in the new form-based configuration
```

File: README.md (hunk 2 above)
Comment (poi33) [reply README.md:9]: This is just the migration. Missing documentation would be a another issue

File: README.md (hunk 1 above)
Comment (poi33) [reply README.md:4]: Nostalgia. ☁️ 
I agree, but that would be upgrading the documentation. Another issue

**Review by rymsha — APPROVED** [review]

**Pull request merged**